libs/tx-statemachine/src/error.test.ts

```diff
@@ -0,0 +1,111 @@
+import { ActionType, BuyAction } from '@thecointech/broker-db';
+import * as Transactions from '@thecointech/broker-db/transaction';
+import { GetContract } from '@thecointech/contract';
+import { getFirestore } from "@thecointech/firestore";
+import { init } from '@thecointech/firestore';
+import { log } from '@thecointech/logging';
+import Decimal from 'decimal.js-light';
+import { DateTime } from 'luxon';
+import * as FSM from '.';
+import { getCurrentState, StateGraph } from './types';
+import { manualOverride, manualOverrideTransition } from './transitions/manualOverride';
+
+const transitionBase = (type: string) =>({
+  timestamp: DateTime.now(),
+  type,
+  // Reset errors/meta
+  error: undefined,
+  meta: undefined,
+});
+// Simple transitions just test the different kind of scenarios the FSM needs to process.
+// noop - no change to data, just transition to a new state
+const noop = async () => transitionBase('noop');
```

Comment:
noop

Reply:
the only state change guaranteed not to crash